snapcraft/internal/lifecycle/_runner.py

```diff
@@ -58,7 +59,7 @@ def execute(step, project_options, part_names=None):
                           over.
     :returns: A dict with the snap name, version, type and architectures.
     """
-    config = snapcraft.internal.load_config(project_options)
+    config = project_loader.load_config(project_options)
```

Comment:
might as well consider unimporting `snapcraft`?

Reply:
and also `snapcraft.internal`

Reply:
We use `snapcraft`, but you're right, `snapcraft.internal` does not seem necessary. Wonder why flake8 didn't grab that. Fixed.